extensions/NV/SPV_NV_tensor_addressing.asciidoc

```diff
@@ -0,0 +1,499 @@
+1+| 4 | 5370 |'Result <id>' | '<id>' +
+'Dim' | '<id>' +
+'ClampMode'
+|=====
+
+[cols="1,1,4*3",width="100%"]
+|=====
+5+|[[OpTypeTensorViewNV]]*OpTypeTensorViewNV* +
+ +
+'Dim' is the number of dimensions in the tensor view, and must be a 
+'constant instruction' with scalar 32-bit 'integer type'. The value must
+be greater than zero and less than or equal to 5. +
+ +
+'HasDimensions' is a boolean indicating whether the view has its own dimensions
+(reinterpreting those from the tensor layout) or if the tensor layout's
+dimensions are used. It must be a 'constant instruction' with scalar
+'boolean type'. +
+ +
+'p0' ... 'p<Dim-1>' are integer values indicating how the tensor's coordinates
+are permuted. They each must be a  'constant instruction' with scalar 32-bit
+'integer type', and they must form a valid permutation of the range [0,Dim). +
+ +
+1+|Capability: +
+*TensorAddressingNV*
+1+| 5+variable | 5371 |'Result <id>' | '<id>' +
+'Dim' | '<id>' +
+'HasDimensions' | '<id>', '<id>', ... +
+'p0', +
+... +
+'p<Dim-1>'
+|=====
+
+3.X Tensor Layout and View Instructions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+New section in 3 "Binary Form".
+
+[cols="1,1,2*3",width="100%"]
+|=====
+3+|[[OpCreateTensorLayoutNV]]*OpCreateTensorLayoutNV* +
+ +
+Create a Tensor Layout of the requested type. The layoutDimension, stride,
+span, and offset elements are initialized to zero. The blockSize elements are
+initialized to one. clampValue is initialized to zero. +
+ +
+'Result Type' must be *OpTypeTensorLayoutNV*. +
+ +
+1+|Capability: +
+*TensorAddressingNV*
+1+| 3 | 5372 | '<id>' +
+'Result Type' | 'Result <id>'
+|=====
+
+[cols="1,1,4*3",width="100%"]
+|=====
+5+|[[OpTensorLayoutSetBlockSizeNV]]*OpTensorLayoutSetBlockSizeNV* +
+ +
+Create a copy of 'TensorLayout', setting the blockSize elements to
+'BlockSize<i>'. When the blockSize is not 1, the strides are considered to be
+in blocks rather than in elements. +
+ +
+The number of BlockSize operands must match the dimension of 'Result Type'. +
+ +
+The BlockSize operands must each be a scalar 32-bit integer type. +
+ +
+The type of 'TensorLayout' must be 'Result Type'. +
+ +
+1+|Capability: +
+*TensorAddressingNV*
+1+| 5+variable | 5384 | '<id>' +
+'Result Type' |'Result <id>' | '<id>' +
+'TensorLayout' | '<id>', '<id>', ... +
+'BlockSize0', +
+... +
+'BlockSize<LDim-1>'
+|=====
+
+[cols="1,1,4*3",width="100%"]
+|=====
+5+|[[OpTensorLayoutSetDimensionNV]]*OpTensorLayoutSetDimensionNV* +
+ +
+Create a copy of 'TensorLayout', setting the layoutDimension and span elements to 'Dim<i>'.
+Sets offset elements to zero. Sets stride[LDim-1] to 1 and sets stride[i] to
+stride[i+1] * ceiling('Dim<i+1>' / blockSize[i+1]). +
+ +
+The number of Dim operands must match the dimension of 'Result Type'. +
+ +
+The Dim operands must each be a scalar 32-bit integer type. +
+ +
+The type of 'TensorLayout' must be 'Result Type'. +
+ +
+1+|Capability: +
+*TensorAddressingNV*
+1+| 5+variable | 5374 | '<id>' +
```

Comment:
```suggestion
1+| 5+variable | 5373 | '<id>' +
```

Reply:
note to self - need to regen html after this change